upload/admin/view/template/module/smaily_for_opencart.tpl

```diff
@@ -209,16 +209,78 @@
         </div>
         <div id="tab-rss" style="display: none">
           <table class="form">
+            <tr>
+              <td>
+                <?php echo $rss_category_title; ?>
+              </td>
+              <td>
+                <select name="smaily_for_opencart_rss_category" id="rss-category" class="smaily-rss-options">
+                    <option value="">All Products</option>
+                    <?php
+                    foreach ($rss_categories as $category) {
+                      $selected = (int) $category['category_id'] === $rss_category ? 'selected' : ''; ?>
+                      <option value="<?php echo $category['category_id']; ?>"<?php echo $selected; ?>><?php echo $category['name']; ?></option>
```

Comment:
Missing whitespace between `value` and `selected` attributes.

A the moment the HTML is rendered as:

```html
<option value="33"selected>Cameras</option>
```

Browsers can handle it, but it is still invalid HTML. Please fix it.

Reply:
Didn't know about this. Will fix it on all versions.